src/clib/pio_darray.c

```diff
@@ -1389,10 +1392,16 @@ int PIOc_write_darray(int ncid, int varid, int ioid, PIO_Offset arraylen, void *
        maximum number of non-contiguous regions in an IO process and
        forcefully flush out user data cached by a compute process
        when that limit has been reached. */
+
+    /* [Update]
```

Comment:
You can remove the [Update] tag here, just add the comments below as continuation of the comments above.

Reply:
OK, I have removed that tag.